src/query-builder/order-by-interface.ts

```diff
@@ -45,24 +45,6 @@ export interface OrderByInterface<DB, TB extends keyof DB, O> {
    * order by "id", "fn" desc
    * ```
    *
-   * Multiple columns/expressions per call:
-   *
-   * ```ts
-   * await db
-   *   .selectFrom('person')
-   *   .select('person.first_name as fn')
-   *   .orderBy(['id', 'fn'])
-   *   .execute()
-   * ```
-   *
-   * The generated SQL (PostgreSQL):
-   *
-   * ```sql
-   * select "person"."first_name" as "fn"
-   * from "person"
-   * order by "id", "fn"
-   * ```
-   *
```

Comment:
we don't want to encourage this deprecated overload's usage. it is a lot slower at compile-time.